extensions/latex/cgmanifest.json

```diff
@@ -6,11 +6,11 @@
 				"git": {
 					"name": "jlelong/vscode-latex-basics",
 					"repositoryUrl": "https://github.com/jlelong/vscode-latex-basics",
-					"commitHash": "45c7b12ee68563afd50407e5eac02d30d33dbe7a"
+					"commitHash": "ff5e3f24ff5f242d62954cf9e124184f27bf24dd"
 				}
 			},
 			"license": "MIT",
-			"version": "1.6.0",
```

Comment:
It looks like the version was changed to 1.5.4, even thought the right [commit](https://github.com/jlelong/vscode-latex-basics/commit/4f8b49bca02b0bfde6e46252c209396ba6c567f9) is referenced below.